module-gui/gui/widgets/text/core/cursors/TextLineCursor.hpp

```diff
@@ -17,6 +17,15 @@ namespace gui
         void handleDownNavigation(unsigned int selectedLineNumber, unsigned int selectedLineCursorPosition);
         void handleUpNavigation(unsigned int selectedLineNumber, unsigned int selectedLineCursorPosition);
 
+        bool handleNextLine() override
+        {
+            return displayNextLine();
+        };
+        bool handlePreviousLine() override
+        {
+            return displayPreviousLine();
+        }
+
```

Comment:
move to cpp :) and [[nodiscard]] ( for the previous comment too )

Reply:
sure